src/plugins/vis_type_vega/public/vega_fn.ts

```diff
@@ -58,9 +58,9 @@ export const createVegaFn = (
     const vegaRequestHandler = createVegaRequestHandler(dependencies, context.abortSignal);
 
     const response = await vegaRequestHandler({
-      timeRange: get(input, 'timeRange'),
-      query: get(input, 'query'),
-      filters: get(input, 'filters'),
+      timeRange: get(input, 'timeRange') as any,
```

Comment:
I think here we can `import { Filter, TimeRange, Query } from '../../data/public';`  and replace `any` with the appropriate type